packages/expo-device/src/ExpoDevice.web.ts

```diff
@@ -15,6 +15,23 @@ function convertGiBtoBytes(gib: number): number {
   return Math.round(gib * 1024 ** 3);
 }
 
+function getDeviceType(): DeviceType {
+  switch (result.device.type) {
```

Comment:
`result` can be null/undefined so probably we should do this 👇 
```suggestion
  switch (result?.device?.type) {
```